src/sentry/replays.py

```diff
@@ -1,10 +1,6 @@
 from __future__ import absolute_import
-
-import socket
-
-from httplib import HTTPConnection, HTTPSConnection
-from urllib import urlencode
-from urlparse import urlparse
+from requests import RequestException
```

Comment:
Surely it is, but ``requests`` also expose ``RequestException`` to ``requests`` module level for convenience.